pkgs/applications/networking/lls/default.nix

```
@@ -0,0 +1,22 @@
+{ rustPlatform, fetchFromGitHub, lib }:
+rustPlatform.buildRustPackage {
+  pname = "lls";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "jcaesar";
+    repo = "lls";
+    rev = "badb937f8c35c2dccc2ae259608cfb606a837f3e";
+    hash = "sha256-Aq0MGhzSoJCkM0Wt/r5JSOz96LyRSgSryD7+m4aFZEA=";
+  };
+
+  cargoSha256 = "sha256-WY4MnPNDWFEzFOehm7TqCL05Ea6n93f8VWBTOuqjBAg=";
+
+  meta = with lib; {
+    description = "lls - list listening sockets";
```

```suggestion
    description = "Tool to list listening sockets";
```

> meta.description should: Not start with the package name.

(see https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes)